service/src/main/java/uk/gov/hmcts/reform/fpl/enums/GeneratedOrderKey.java

```diff
@@ -10,7 +10,8 @@
     ORDER("order"),
     JUDGE_AND_LEGAL_ADVISOR("judgeAndLegalAdvisor"),
     ORDER_FURTHER_DIRECTIONS("orderFurtherDirections"),
-    ORDER_MONTHS("orderMonths");
+    ORDER_MONTHS("orderMonths"),
+    INTERIM_END_DATE("interimEndDate");
```

Comment:
Do we need a new key for these fields perhaps? They are no longer generic order properties.

Reply:
InterimOrderKey?

Reply:
Yeah sounds good